libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -322,6 +322,16 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     AP_GROUPINFO("COM_PORT2", 24, AP_GPS, _com_port[1], 1),
 #endif
 
+#if GPS_MAX_RECEIVERS > 1
+    // @Param: PRIMARY
+    // @DisplayName: Primaray GPS
+    // @Description: This GPS will be used preferentially
+    // @Increment: 1
+    // @Values: 0:GPS 1, 1:GPS 2
```

Comment:
To be consistent with AP_Baro the values could be "FirstGPS" and "SecondGPS". https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Baro/AP_Baro.cpp#L113

Reply:
done